pkg/templates/aws/edges/eks_node_group-eks_cluster.yaml

```diff
@@ -23,85 +23,42 @@ operational_rules:
               Chart: metrics-server
               Internal: true
 
-  - if: | # check if there is an aws-cloudwatch namespace so we can install fluent bit in there
+  - if: | # check if there is an eks add on for cloudwatch so we can install observability
```

Comment:
I don't understand EKS enough to review this template, seems okay from a high-level though